src/main/java/io/openliberty/tools/intellij/lsp4mp4ij/psi/core/java/diagnostics/IJavaDiagnosticsParticipant.java

```diff
@@ -24,6 +25,8 @@
  */
 public interface IJavaDiagnosticsParticipant {
 
+	public static final ExtensionPointName<IJavaDiagnosticsParticipant> EP_NAME = ExtensionPointName.create("open-liberty.intellij.javaDiagnosticsParticipant");
```

Comment:
This is unused. The real extension point is in `JavaDiagnosticsDefinition` which implements `IJavaDiagnosticsParticipant.

Reply:
Added comment for the benefit of future maintainers.